actions/class.ItemContent.php

```diff
@@ -21,6 +21,7 @@
 
 use oat\generis\model\OntologyAwareTrait;
 use oat\tao\helpers\FileUploadException;
+use oat\tao\model\accessControl\Context;
```

Comment:
AFAIK this has been added at core `v48.22.1`

Reply:
I will update with new tao-core version after this PR: https://github.com/oat-sa/tao-core/pull/3048